Contest_API.md

```diff
@@ -1561,10 +1562,10 @@ The following endpoints are associated with clarification messages:
 
 | Endpoint                             | Mime-type        | Required? | Description
 | :----------------------------------- | :--------------- | :-------- | :----------
-| `/contests/<id>/clarifications`      | application/json | yes       | JSON array of all clarification messages with elements as defined in the table below.
-| `/contests/<id>/clarifications/<id>` | application/json | yes       | JSON object of a single clarification message with elements as defined in the table below.
+| `/contests/<id>/clarifications`      | application/json | yes       | JSON array of all clarifications with properties as defined in the table below.
+| `/contests/<id>/clarifications/<id>` | application/json | yes       | JSON object of a single clarification with properties as defined in the table below.
 
-JSON elements of clarification message objects:
+Properties of clarification objects:
```

Comment:
I don't agree with the dropping of "message".

A clarification request is not a clarification, but both are (arguably) clarification messages.

Reply:
I was on the fence because the object is called 'clarification', not 'clarification-message', but I've reverted this in the next commit.